### PR TITLE
Fix typos in Go tests

### PR DESCRIPTION
**Pull request by alexandear**
This PR fixes various spelling mistakes in tests.

### CODE REVIEW

File: config/tests/jobs/jobs_test.go [hunk 1]
```diff
@@ -706,7 +706,7 @@ func checkKubekinsPresets(jobName string, spec *coreapi.PodSpec, labels map[stri
 			}
 		}
 
-		if scenario == "kubenetes_e2e" {
+		if scenario == "kubernetes_e2e" {
```
Comment (petr-muller) [inline config/tests/jobs/jobs_test.go:709]: this one looks like it may have a counterpart somewhere?

Comment (alexandear) [reply config/tests/jobs/jobs_test.go:709]: Yes, we have file https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_e2e.py

Comment (petr-muller) [reply config/tests/jobs/jobs_test.go:709]: Does that mean this job works incorrectly now until the typo is fixed, or that something relies on that fix? It looks like the former...

**Pull request closed without merging**